ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -644,7 +644,7 @@ tests:
         make undeploy9-runtimes-pytorch-ubi9-python-3.11
       dependencies:
       - env: RUNTIME_PYTORCH_IMAGE
-        name: runtime-pytorch-ubi9-python-3.11
+        name: runtime-cuda-pytorch-ubi9-python-3.11
```

Comment:
That would be

https://github.com/openshift/release/blob/a9c7af41fe366038ce0600fda193e2d8ccf2d614/ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml#L31-L33

/lgtm